src/uct/ib/base/ib_md.c

```diff
@@ -1624,6 +1702,13 @@ static ucs_status_t uct_ib_verbs_md_open(struct ibv_device *ibv_device,
         goto err_free_context;
     }
 
+    uct_ib_md_parse_relaxed_order(md, md_config);
+    if (md->relaxed_order) {
+        md->memh_struct_size = sizeof(uct_ib_verbs_mem_t);
+    } else {
+        md->memh_struct_size = sizeof(uct_ib_verbs_mem_t) - sizeof(uct_ib_mr_t);
+    }
```

Comment:
how come common code looks at "verbs" struct?
this calculation may be wrong for mlx5

Reply:
it's verbs code

Reply:
it's src/uct/ib/**base**/ib_md.c

Reply:
right, and it hosts uct_ib_verbs_md_ops

Reply:
right, it's uct_ib_**verbs**_md_open
maybe define uct_ib_verbs_mem_t with zero-size array of MRs, so it will be `sizeof(uct_ib_verbs_mem_t) + sizeof(uct_ib_mr_t)`, i think it's more obvious (+add comment to explain the calculation)